test/markdownlint-test.js

```diff
@@ -1059,6 +1097,20 @@ tape("configBadChildFile", (test) => {
     });
 });
 
+tape("configBadChildPackage", (test) => {
+  test.plan(4);
+  markdownlint.readConfig("./test/config/config-badchildpackage.json",
+    function callback(err, result) {
+      test.ok(err, "Did not get an error for bad child file.");
+      test.ok(err instanceof Error, "Error not instance of Error.");
+      // @ts-ignore
+      test.equal(err.code, "ENOENT",
```

Comment:
In this case of requiring a module, I think `MODULE_NOT_FOUND` is a better code (per discussion above).

Reply:
What about `@my-company/my-package/my-file.json` (assuming the package exists, but the file within it does not)?

Reply:
I was not meaning to suggest adding special error handling, just that if the new behavior gives a better error, that is nice!